tests/cases/string/contains.test

```diff
@@ -0,0 +1,14 @@
+### SUBSTRAIT_SCALAR_TEST: v1.0
+### SUBSTRAIT_INCLUDE: '/extensions/functions_string.yaml'
+
+# basic: Basic examples contains as prefix
+contains('abcdefg'::str, 'abc'::str) = true::bool
+contains('abcdefg'::str, 'CdE'::str) [case_sensitivity:CASE_INSENSITIVE] = true::bool
```

Comment:
would also be useful to test what happens when case insensitivity is not specified for the same input